src/main/scala/no/ndla/searchapi/service/search/LearningPathIndexService.scala

```diff
@@ -75,7 +75,12 @@ trait LearningPathIndexService {
           ),
           intField("isBasedOn"),
           keywordField("supportedLanguages"),
-          getTaxonomyContextMapping
+          getTaxonomyContextMapping,
+          nestedField("embedResourcesAndIds").fields(
+            keywordField("resource"),
+            keywordField("id"),
+            keywordField("language")
+          )
```

Comment:
Denne vil aldri ha innhold. Må den være med for at søket ikkje skal feile?

Reply:
Sant, jeg stusset litt på dette selv. Søket feilet fram til jeg la til disse linjene. Det finnes kanskje en annen måte å omgå dette problemet på?